packages/react-scripts/package.json

```diff
@@ -60,8 +65,16 @@
     "whatwg-fetch": "2.0.3"
   },
   "devDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "bootstrap": "^4.0.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "jest-context": "^2.1.0",
```

Comment:
Getting an output warning with yarn on this one with: `quickstart-react-scripts > jest-context@2.1.0: jest-context is now a jest plugin. Please install jest-plugin-context instead.`

Reply:
I can make that change, thanks